experimental/dds/tree2/src/shared-tree/sharedTreeView.ts

```diff
@@ -121,13 +138,17 @@ export interface ISharedTreeView extends AnchorLocator {
 	readonly editor: IDefaultEditBuilder;
 
 	/**
-	 * Undoes the last completed transaction made by the client.
+	 * Undoes the last completed transaction made by the client. Calling this does nothing if there are no transactions in the
+	 * undo stack.
```

Comment:
nit: The new sentence you added, as well as the existing sentence below the line-break below, seem like good candidates for going under a "remarks" block.

Same for the redo API.

Reply:
Good call.